### PR TITLE
Docs: update packaging and installation instructions

### PR DESCRIPTION
**Pull request by ngetahun**
Add helm-chart installation documentation

## Description

Please describe your change and provide as much context as possible.

Fixes # (issue)

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [x] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [x] I have verified that my code follows RMT's coding standards with `rubocop`.
- [x] I have reviewed my own code and believe that it's ready for an external review.
- [x] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [x] RMT's test coverage remains at 100%.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/298768/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged a217ee0017c270a99779fa8b05e096a58cb614c8 into e54da1b4500fddaa684869a1204a9d425b11154d)

File: PACKAGE.md [hunk 1]
```diff
@@ -100,3 +102,10 @@ Using target project 'SUSE:Maintenance'
 You can check the status of your requests [here](https://build.opensuse.org/package/requests/systemsmanagement:SCC:RMT/rmt-server) and [here](https://build.suse.de/package/requests/Devel:SCC:RMT/rmt-server).
 
 After your requests have been accepted, they still have to pass maintenance testing before they are released to customers. You can check their progress at [maintenance.suse.de](https://maintenance.suse.de/search/?q=rmt-server). If you still need help, the maintenance team can be reached at [maint-coord@suse.de](maint-coord@suse.de) or #maintenance on irc.suse.de.
+
+
+## Packaging and publishing to SUSE registry
+
+SUSE registry houses the rmt-server docker image. The image is built on OBS/IBS, project for SLES 15sp4 based distributions can be found [here](https://build.opensuse.org/package/show/devel:BCI:SLE-15-SP4/rmt-server-image).
+
+At the moment of writing, the publishing process has to be done manually. This can be achieved my reaching out to the Auto-Build team (only available internally).
```
Comment (digitaltom) [inline PACKAGE.md:111]: Can we add a pointer here how to reach the Autobuild team?

Comment (ngetahun) [reply PACKAGE.md:111]: It's in the `friends-of-scc` docs. I left it obsecure as this is a public facing repo

Comment (digitaltom) [reply PACKAGE.md:111]: ok

**Review by digitaltom — APPROVED** [review]

**Pull request merged**